### PR TITLE
cucushift/installer/check/sshkey: check root user could not login node

### PR DESCRIPTION
**Pull request by jinyunma**
https://redhat-internal.slack.com/archives/CF8SMALS1/p1729047124776669

Add checkpoint that root user could not login node with all configured ssh private keys

### CODE REVIEW

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.17-azure-ipi-private-sshkey-f14

File: ci-operator/step-registry/cucushift/installer/check/sshkey/cucushift-installer-check-sshkey-commands.sh [hunk 1]
```diff
@@ -72,15 +73,37 @@ fi
...
+        SSH_KEY_FILES="${SSH_KEY_FILES} ${SHARED_DIR}/key-${key_type}"
+    else
+        echo "ERROR: could not find private sshkey for key type '${key_type}' in '${SHARED_DIR}'!"
+        exit 1
+    fi
+done
+
+#Append default ssh key specified in install-config
+#make sure that user root could not login with any configured ssh key
+SSH_KEY_FIELS="${SSH_KEY_FIELS} ${DEFAULT_SSH_PRIV_KEY_PATH}"
```
[the remark this answers is not in the document]
Comment (gpei) [reply ci-operator/step-registry/cucushift/installer/check/sshkey/cucushift-installer-check-sshkey-commands.sh:89]: Or I think we can just declare SSH_KEY_FILES="${DEFAULT_SSH_PRIV_KEY_PATH}" in Line#76 as it's the default ssh private key file in install-config, then we don't need this Line#89 to append it.

[the remark this answers is not in the document]
Comment (jinyunma) [reply ci-operator/step-registry/cucushift/installer/check/sshkey/cucushift-installer-check-sshkey-commands.sh]: good points, updated.

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.17-azure-ipi-private-sshkey-f14

**Pull request merged**